### PR TITLE
36 mock data

### LINKED ISSUE #36: create mock data and types
Database structure is in [draw.io](https://drive.google.com/file/d/1EcEML3q8ObomGcZjH0liFS6zKG7V7-fc/view?usp=sharing)

- create mockdata for Location, User, Venue and Activity.

### PR DESCRIPTION
**Pull request by DLittlefield81**
chore: create mock data

### CODE REVIEW

**Comment by rmc3408** [comment]
Complete those fields
image

File: src/data/mockData.ts [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+]
+
+export const activityData: Activity[] = [
+  {
+    name: 'play game 1',
+    description: 'write something about the activity',
+    image: 'http://www.playgame.com/image1',
+    startTimes: [1692349200000, 1692381600000],
+    duration: 45,
+    category: <Category>'Indoor',
```
Comment (rmc3408) [inline src/data/mockData.ts]: In category, this is the data model for PRISMA on backend. Change for example: ['a','b']

File: src/data/mockData.ts [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    address: '4 Water Road',
+    phone: '1115433366',
+    city: 'Mississauga',
+    postalCode: 'L5H 1R1',
+    latitude: 12,
+    longitude: -39,
+  },
+]
+
+export const userData: User[] = [
```
Comment (rmc3408) [inline src/data/mockData.ts]: Run the backend server and get the response example from `http://localhost:3001/users`

File: src/data/mockData.ts [hunk 3]
```diff
@@ -0,0 +1,169 @@
+export interface Location {
```
Comment (rmc3408) [inline src/data/mockData.ts]: move to all types to `types` folder

**Review by rmc3408 — COMMENTED** [review]
few changes

File: src/data/mockData.ts (hunk 2 above)
Comment (DLittlefield81) [reply src/data/mockData.ts]: Cannot connect to backend. 
tried with assistance.

File: src/data/mockData.ts (hunk 1 above)
Comment (DLittlefield81) [reply src/data/mockData.ts]: okay

File: src/data/activitiesMockData.ts [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import { Activity } from '../types/activityType'
```
Comment (rmc3408) [inline src/data/activitiesMockData.ts]: try use -> import { type Activity } --- just make clear.

File: src/types/userType.ts [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import { Venue } from './venueType'
```
Comment (rmc3408) [inline src/types/userType.ts:1]: import { type Venue }

File: src/data/usersMockData.ts [hunk 6]
```diff
@@ -0,0 +1,351 @@
+import { User } from '../types/userType'
```
Comment (rmc3408) [inline src/data/usersMockData.ts]: import { type USer }

File: src/data/locationsMockData.ts [hunk 7]
```diff
@@ -0,0 +1,31 @@
+import { Location } from '../types/locationType'
```
Comment (rmc3408) [inline src/data/locationsMockData.ts]: import { type Location

**Review by rmc3408 — APPROVED** [review]
I think is pretty good.

If you wanna make better use `import { type X }`

Since they are in the folder types. Please, change filename from `userType.ts` to just `user.ts`

but keep for userMockData... because in this folder will have more data in the future. not just mock.

**Pull request merged**